### PR TITLE
encryption: added required playback policy callout on encrypted vod

### PR DESCRIPTION
**Pull request by gioelecerati**
## Description

_Concise description of proposed changes_

### CODE REVIEW

File: pages/guides/developing/encrypted-vod.en-US.mdx [hunk 1]
```diff
@@ -101,7 +101,7 @@ Now, we can use asymmetric encryption with the Livepeer Public Key to encrypt ou
 
 ```js
 // Decode the SPKI public key from base64 and convert it to a buffer
-spkiPublicKey = atob(publicKeyData.spki_public_key);
+const spkiPublicKey = atob(publicKeyData.spki_public_key);
 const publicKeyBuffer = Uint8Array.from(atob(spkiPublicKey), (c) =>
```
Comment (victorges) [inline pages/guides/developing/encrypted-vod.en-US.mdx:105]: Isn't `spkiPublicKey` var already `atob`'ed? Do we have some double b64 encoding somewhere?

File: pages/guides/developing/encrypted-vod.en-US.mdx [hunk 2]
```diff
@@ -140,6 +140,10 @@ We're all set to upload the video file.
 When requesting an upload to the Livpeer API, you can simply add an `encryption` field with your encrypted key.
 [Api Reference](https://docs.livepeer.org/reference/api/create-asset-via-url)
 
+<Callout>
+  When uploading an encrypted video, specifying a `playbackPolicy` is required.
```
Comment (victorges) [inline pages/guides/developing/encrypted-vod.en-US.mdx:144]: Maybe worth noting that you can still specify it as `public` if that's what you want, but beware that anyone with the `playbackId` or `playbackUrl` would be able to watch it

**Review by victorges — APPROVED** [review]
either way

File: pages/guides/developing/encrypted-vod.en-US.mdx (hunk 1 above)
Comment (gioelecerati) [reply pages/guides/developing/encrypted-vod.en-US.mdx:105]: Subtle crypto want the actual key atobed, so we need to atob it again

**Pull request merged**